src/article.h

```diff
@@ -0,0 +1,107 @@
+#pragma once
+
+#include <napi.h>
+#include <zim/article.h>
+#include <zim/blob.h>
+#include <zim/writer/creator.h>
+
+#include <memory>
```

Comment:
@kelson42 `article.h` includes `memory` but CodeFactor is saying it doesn.t

Reply:
@kerlvinHammond I think we should just ignore CodeFactor here.